test/testutil.h

```diff
@@ -402,4 +402,48 @@ void test_openssl_errors(void);
 extern BIO *bio_out;
 extern BIO *bio_err;
 
+/*
+ * Utilities to parse a test file.
+ */
+#define TESTMAXPAIRS        20
+
+typedef struct pair_st {
+    char *key;
+    char *value;
+} PAIR;
+
+typedef struct stanza_st {
+    const char *test_file;      /* Input file name */
+    BIO *fp;                    /* Input file */
+    char buff[10240];           /* Input buffer */
```

Comment:
And to answer the real question, it's how big a single key/value pair can be.  Could be reduced to 4096 and everything still passes :)